packages/cli/lib/project-generator.js

```diff
@@ -177,11 +190,11 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       },
     ];
     return this.prompt(prompts).then(props => {
-      const settings = props.settings || choices.map(c => c.name);
+      const settings = props.settings || choices.map(c => c.name.split(':')[0]);
```

Comment:
I find this string manipulation rather hacky. Isn't the original option name available as `c.key`?

Reply:
It has to include the word `Enable` but I'll replace it with `Enable ${c.key}` so it's less hacky.